packages/core/src/node/backend-application.ts

```diff
@@ -169,13 +174,21 @@ export class BackendApplication {
         process.on('SIGPIPE', () => {
             console.error(new Error('Unexpected SIGPIPE'));
         });
-
+        /**
+         * Kill the current process tree on exit.
+         */
+        function signalHandler(signal: NodeJS.Signals): never {
+            const code = process.platform === 'win32'
+                ? 1 // Windows always seem to exit with 1
+                : 128 + (os.constants.signals[signal] ?? -127);
```

Comment:
This may need more comment than the `1` for Windows :-).

Reply:
Right, I simplified it to always 1 for now.